### PR TITLE
Feature/uai email lazy reviewers

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

File: venues/auai.org/UAI/2017/python/email-lazy-reviewers.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+verbose = True if args.verbose and args.verbose.lower()=='true' else False
+
+
+total_missing = 0;
+total_complete = 0;
+
+def get_data(invitation):
+
+    headers = {'User-Agent': 'test-create-script', 'Content-Type': 'application/json', 'Authorization': 'Bearer ' + client.token}
+    anon_reviewers = requests.get(client.baseurl+'/groups?id=auai.org/UAI/2017/Paper.*/AnonReviewer.*', headers = headers)
```
Comment (mspector) [inline venues/auai.org/UAI/2017/python/email-lazy-reviewers.py]: I guess you can't use the client.get_groups() method for this?

If that doesn't work, I can add it to my to-do list for openreview-py

File: venues/auai.org/UAI/2017/python/email-lazy-reviewers.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+        if note_id:
+            total_complete += 1
+        else:
+            total_missing += 1
+            if verbose:
+                print "late reviewers on paper %s: %s" % (paper_number, reviewer)
+            late_users.add(reviewer)
+
+# Comment this out to send the emails
```
Comment (mspector) [inline venues/auai.org/UAI/2017/python/email-lazy-reviewers.py]: not that important, but I guess this should be "uncomment this out to send the emails"

File: venues/auai.org/UAI/2017/python/email-lazy-reviewers.py (hunk 1 above)
Comment (melisabok) [inline venues/auai.org/UAI/2017/python/email-lazy-reviewers.py]: No, there is not method in openreview-py to do this.

Comment (mspector) [inline venues/auai.org/UAI/2017/python/email-lazy-reviewers.py]: ok, thanks

**Pull request merged**